### PR TITLE
Introduce env variable ZOO_4LW_COMMANDS_WHITELIST

### PR DESCRIPTION
**Pull request by vistrcm**
Introduce environment variable ZOO_4LW_COMMANDS_WHITELIST to change configuration parameter `4lw.commands.whitelist cluster`.

Possible solution of request #76

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+zookeeper-docker.iml
```
Comment (31z4) [inline .gitignore]: Please keep this in your local gitignore because it's specific to your environment.

**Comment by 31z4** [comment]
Hello @vistrcm, thanks for your interest in contributing to the image 👍 I'm happy to merge this once you address one comment 🙂

File: .gitignore (hunk 1 above)
Comment (vistrcm) [reply .gitignore]: Good catch! Pull Request updated.
Thanks for letting me know.

**Comment by 31z4** [comment]
https://github.com/docker-library/official-images/pull/6564
https://github.com/docker-library/docs/pull/1564

**Pull request merged**